module/notification/src/Domain/Query/NotificationQueryInterface.php

```diff
@@ -23,7 +24,12 @@ public function getDataSet(UserId $id, Language $language): DataSetInterface;
     public function check(UserId $id): array;
 
     /**
-     * @throws \Doctrine\DBAL\DBALException
+     * @throws DBALException
```

Comment:
This way you are coupling interface with the DBAL - the thing that interfacing tries to avoid. If desired to be annotated(not saying it's a bad idea) domain(SharedKernel probably) exception needed